### PR TITLE
style: run prettier on web/api/x and css

### PR DESCRIPTION
**Pull request by nschonni**
Not run on code fences

### CODE REVIEW

File: files/en-us/web/api/xrframe/getdepthinformation/index.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ An {{domxref("XRCPUDepthInformation")}} object.
 - `NotSupportedError` {{domxref("DOMException")}}
   - : Thrown if `"depth-sensing"` is not in the list of enabled features for this {{domxref("XRSession")}}.
```
Comment (teoli2003) [inline files/en-us/web/api/xrframe/getdepthinformation/index.md:38]: Odd. Why no extra lines here, but at the next line?

File: files/en-us/web/api/xsltprocessor/basic_example/index.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ To try out the example:
 
    > **Warning:** Opening the XML file directly from the file system will not work, because loading the stylesheet from the file system is a [cross-origin request](/en-US/docs/Web/HTTP/CORS), and will be disallowed by default.
    > Hosting the XML and stylesheet on the same local server ensures that they have the same origin.
+
 1. Open **example.xml** from the browser.
 1. The browser output is then as shown below:
```
Comment (teoli2003) [inline files/en-us/web/api/xsltprocessor/basic_example/index.md:64]: Side note: We should consider fixing and activating MD029. (Not in this PR, of course)

File: files/en-us/web/api/xrframe/getdepthinformation/index.md (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/api/xrframe/getdepthinformation/index.md:38]: I think the parser tries to guess the format of the list, and sometimes it chooses oddly. In some of the previous PRs I had deleted the extra lines to make it happy, but I just mostly took the defaults in these ones

File: files/en-us/web/css/_doublecolon_-moz-range-thumb/index.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ input[type=range]::-moz-range-thumb {
 
 A progress bar using this style might look something like this:
 
-![The thumb of the \<input type=right> styled in green](screen_shot_2015-12-04_at_13.30.08.png)
+![The thumb of the <input type=right> styled in green](screen_shot_2015-12-04_at_13.30.08.png)
```
Comment (teoli2003) [inline files/en-us/web/css/_doublecolon_-moz-range-thumb/index.md]: So this is https://github.com/DavidAnson/markdownlint/issues/579.

Even if I agree that Markdownlint shouldn't flag it, considering that we are automating Markdownlint but not (yet) Prettier, we should keep the Markdownlint version for now. 

There are few occurrences (though we will likely add some as we add more _alt_ texts these days). But this is a problem we will need to solve before activating Prettier.

Let's keep the Markdownlint-compliant way for now.

```suggestion
![The thumb of the \<input type=right> styled in green](screen_shot_2015-12-04_at_13.30.08.png)
```

File: files/en-us/web/css/css_animations/using_css_animations/index.md [hunk 4]
```diff
@@ -45,10 +46,8 @@ The sub-properties of the {{cssxref("animation")}} property are:
   - : Specifies whether to pause or play an animation sequence.
 - {{cssxref("animation-timing-function")}}
   - : Specifies how an animation transitions through keyframes by establishing acceleration curves.
-<!--
 - {{cssxref("animation-timeline")}}
   - : Specifies the names of one or more {{cssxref("@scroll-timeline")}} at-rules describing the scroll animations.
```
Comment (teoli2003) [inline files/en-us/web/css/css_animations/using_css_animations/index.md:50]: The intent of the comment is to hide what is in between. Let's remove it.
```suggestion
```

Strangely, this change didn't happen at line 28…

File: files/en-us/web/css/path/index.md [hunk 5]
```diff
@@ -33,9 +33,9 @@ path([<'fill-rule'>,]?<string>)
 
 - `<'fill-rule'>`
   - : The filling rule for the interior of the path.
```
Comment (teoli2003) [inline files/en-us/web/css/path/index.md:35]: Again, I don't understand why Prettier doesn't add an empty line here, while it does it sometimes.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/_doublecolon_-moz-range-thumb/index.md (hunk 3 above)
Comment (teoli2003) [reply files/en-us/web/css/_doublecolon_-moz-range-thumb/index.md]: You just did it.

File: files/en-us/web/api/xrframe/getdepthinformation/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/xrframe/getdepthinformation/index.md:38]: Yeah, it looks like it tries to have symmetry (if an empty line after, it adds an empty line before)

I'm fine with this.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**